src/utils/config/risk-framework.json

```diff
@@ -854,7 +854,11 @@
                     "veLp_arb_Solidex",
                     "fbeets compounder",
                     "genlevgeist",
-                    "GeistLenderBorrower"
+                    "GeistLenderBorrower",
```

Comment:
Also here.

Reply:
Resolved in https://github.com/yearn/yearn-watch/pull/281/commits/ed4fdc112d8b4aad30cc53acaf3687df1b95334b